src/virtual-scroll.ts

```diff
@@ -160,10 +167,12 @@ export class VirtualScrollComponent implements OnInit, OnChanges, OnDestroy {
 
   private addParentEventHandlers(parentScroll: Element | Window) {
     if (parentScroll) {
-      parentScroll.addEventListener('scroll', this.refreshHandler);
-      if (parentScroll instanceof Window) {
-        parentScroll.addEventListener('resize', this.refreshHandler);
-      }
+      this.zone.runOutsideAngular(() => {
+        parentScroll.addEventListener('scroll', this.refreshHandler);
```

Comment:
Use `Renderer2` instead of direct DOM manipulation.

Reply:
Will do

Reply:
Done.